packages/e2e/local-network/scripts/make-babbage.sh

```diff
@@ -273,14 +275,24 @@ mkdir -p ./config/network/genesis/
 cp ./templates/babbage/db-sync-config.json ./config/network/cardano-db-sync/config.json
 cp ./templates/babbage/node-config.json ./config/network/cardano-node/config.json
 
-$SED -i -E "s/\"ByronGenesisHash\": \".*\"/\"ByronGenesisHash\": \"${byronGenesisHash}\"/" ./config/network/cardano-node/config.json
-$SED -i -E "s/\"ShelleyGenesisHash\": \".*\"/\"ShelleyGenesisHash\": \"${shelleyGenesisHash}\"/" ./config/network/cardano-node/config.json
-$SED -i -E "s/\"AlonzoGenesisHash\": \".*\"/\"AlonzoGenesisHash\": \"${alonzoGenesisHash}\"/" ./config/network/cardano-node/config.json
+###############################
+# NOTE: We can't use -i to do inline replacement because this breaks due to permission errors on MacOS when creating the temp output file in this case, giving Root permissions to Docker is also not 
```

Comment:
This comment is a great note for the person who is doing a code review, additional context to reason about the change. 

I would prefer to see it as a comment in Github UI though, as you put focus on what cannot be done / be used. Not on what you have actually decided to do.

I like the content you have put in PR's **Proposed solution**.

`Use a temporary file to make the changes and copy them back to where we want the change to be.`

Clean and straight.

If you like to keep this link to the forum or leave more context you can extend it ex 

```
###############################
# Use a temporary file to make the changes and copy them back to where we want
# the change to be, as we can't use -i to do inline replacement because this breaks 
# due to permission errors on MacOS
#
# https://forums.docker.com/t/sed-couldnt-open-temporary-file-xyz-permission-denied-when-using-virtiofs/125473
###############################
```

What do you think about it @VanessaPC?

btw. We can slightly improve this comment by limiting lines' length to about 80 chars.

Reply:
Yeah, that sounds a good idea, I'll do the changes, thank you